mlir/lib/Dialect/Affine/Transforms/LoopCoalescing.cpp

```diff
@@ -35,13 +35,17 @@ namespace {
 struct LoopCoalescingPass
     : public affine::impl::LoopCoalescingBase<LoopCoalescingPass> {
 
+  void getDependentDialects(DialectRegistry &registry) const override {
+    registry.insert<affine::AffineDialect>();
+  }
```

Comment:
Why is this necessary? Can this pass produce entities from the affine dialect when none were present in the input?

Reply:
Removed now.